ting.admin.inc

```diff
@@ -162,6 +162,22 @@ function ting_admin_ranking_settings($form, &$form_state) {
     ),
   );
 
+
+  $ting_sort_value = variable_get('ting_sort_default', 'rank_frequency');
+  $form['ting_sort_default'] = array(
+    '#title' => t('Default sort method (best match)'),
+    '#type' => 'fieldset',
+    '#prefix' => '<div id="sort-default-fields-wrapper">',
+    '#suffix' => '</div>',
+    '#tree' => TRUE,
+  );
+
+  $form['ting_sort_default']['fields'] = array(
+    '#type' => 'ting_sort_default',
```

Comment:
Is there any reason why you have chosen to create a special form field type here? It does not seem very complex to me.